sonata-project/page-bundle/4.9/post-install.txt

```diff
@@ -0,0 +1,7 @@
+ * Your Page is almost ready:
+    1. Generate a migration for Sonata Page Database <info>php bin/console doctrine:migrations:diff</info>
+    2. Apply the migration generated <info>php bin/console doctrine:migrations:migrate --no-interaction</info>
+    3. Create your first site: <info>bin/console sonata:page:create-site --enabled --name=localhost --locale=- --host=localhost --relativePath=- --enabledFrom=now --enabledTo=- --default</info>
```

Comment:
Do you need to remove `php` part as well as for 4.0 post-install.txt?

```suggestion
    1. Generate a migration for Sonata Page Database <info>bin/console doctrine:migrations:diff</info>
    2. Apply the migration generated <info>bin/console doctrine:migrations:migrate --no-interaction</info>
    3. Create your first site: <info>bin/console sonata:page:create-site --enabled --name=localhost --locale=- --host=localhost --relativePath=- --enabledFrom=now --enabledTo=- --default</info>
```

Reply:
Hi @diimpp , thank you to Review, 

It works with both ways, but I'm removing because I want to keep like in the docs 

https://github.com/sonata-project/SonataPageBundle/pull/1809/files#diff-616dbd37597845318190158bb507fa469a0edcf22ef874936b54972be8d27a0dR19

And the doc's lint doesn't allow to use php as command prefix